databricks/koalas/namespace.py

```diff
@@ -738,9 +743,10 @@ def read_excel(
     Parameters
     ----------
     io : str, file descriptor, pathlib.Path, ExcelFile or xlrd.Book
-        The string could be a URL. Valid URL schemes include http, ftp, s3,
-        gcs, and file. For file URLs, a host is expected. For instance, a local
-        file could be /path/to/workbook.xlsx.
+        The string could be a URL. If the underlying Spark is above 3.0.0, the value URL must
+        be available by the Spark; otherwise the valid URL schemes include http, ftp, s3, gcs,
```

Comment:
@ueshin, what about we just throw an exception if we have to delegate it to pandas? We could say like "use ks.from_pandas(pd.read_excel(...)) as a workaround".

Reply:
I pushed a commit. I hope it's addressing what you meant.